### DISCUSSION TITLE
WSD500A no battery level information

### DISCUSSION THREAD

**Discussion by AlexValery13** [Koenkk/zigbee2mqtt#10593] [Devices]
Zigbee2mqtt version: 1.22.2-1
[The WSD500A sensor is correctly detected,](https://disk.yandex.ru/d/oILiKIzgkKdd-A) 
[but no battery level information.](https://disk.yandex.ru/d/LCDPvnei6zN5Fw)
[The Smart Life app shows the battery level correctly.](https://disk.yandex.ru/d/LJq3pCkL4t2MTw)
I removed the WSD500A sensor and reconnected. I even used a fresh clean install of HA but the result is the same.
What am I doing wrong :) ?

**Comment by barumbek** [comment]
I have an identical problem. Any help guys.

**Comment by AlexValery13** [comment]
If I briefly press the pairing button immediately after detection, then the battery information appears. I don't think this is a normal solution, but it works.

**Reply by barumbek** [reply]
It worked. And yes, it's really weird. Since through the application it works without additional steps.

**Reply by meijron** [reply]
Press once reset after pairing worked for me as well to get the battery showing in Zigbee2mqtt, thanks for chairing.
(it shows 100% battery with new battery but is does not show millivolts of battery, will see if it goes down in couple of week/month)

**Reply by Ev15** [reply]
Same here. Thanks .

**Reply by evlo** [reply]
very weird, but it at least shows something
Using same trick for different device with this issue i did get 94% and not 100 so maybe it actually works.

I have created PR to add this to docu https://github.com/Koenkk/zigbee2mqtt.io/pull/1546

**Comment by radek-kafka** [comment]
Please when exactly I have to briefly reset the device ? I have tried many times and did not succeed with it .. It is almost "cosmetic" problem till batteries will be almost empty and I will not know about till they are empty ..
